test/edu-benefits/1990e/config/benefitSelection.unit.spec.jsx

```diff
@@ -16,9 +16,8 @@ describe('Edu 1990e benefitEligibility', () => {
           data={{}}
           uiSchema={uiSchema}/>
     );
-    const fields = ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input');
-
-    expect(fields.length).to.equal(2);
+    expect(ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input'))
```

Comment:
Why did you make this change? If it was causing a test failure, I think you may need to run `npm install`.